modules/network/internet_gateway.tf

```diff
@@ -9,6 +13,10 @@ resource "aws_route_table" "internet_gateway" {
     cidr_block = "0.0.0.0/0"
     gateway_id = aws_internet_gateway.main.id
   }
+
+  tags = {
+    Name = "Spacelift Internet Gateway Route Table (${var.suffix})"
+  }
```

Comment:
I think this'll override the `default_tags`. shouldn't we merge the two?

Reply:
I don't think so - I think it handles that automatically:

<img width="689" alt="image" src="https://github.com/user-attachments/assets/ff025611-b649-49ee-8b25-6c34522947a3" />

Here's the docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs#default_tags-configuration-block.